DXRCore/DeusEx/Classes/DXRMenuReSetupRando.uc

```diff
@@ -0,0 +1,36 @@
+class DXRMenuReSetupRando extends DXRMenuSetupRando;
+
+event InitWindow()
+{
+    Super.InitWindow();
+    GetDxr();
+    Init(GetDxr());
+}
+
+function DXRando GetDxr()
+{
+    if( dxr != None ) return dxr;
+    dxr = class'DXRando'.default.dxr;
+    flags = dxr.flags;
+    return dxr;
+}
+
+//If changing ranges in this menu, make sure to update any clamped ranges in DXRFlags ScoreFlags function to match
+function BindControls(optional string action)
+{
+    if(action=="BACK") return;
+
+    //NewGroup("Test"); // TODO: clean way to replicate options from basic new game screen: loadout, difficulty, autosave, crowd control, and mirrored maps
```

Comment:
actually difficulty preset wouldn't make sense here because it would overwrite every other field